libs/ngx-voice/modules/web/intent/intent.service.ts

```diff
@@ -0,0 +1,57 @@
+import { Injectable } from '@angular/core';
+import { NgxVoiceIntent } from '../../core/services/intent.interface';
```

Comment:
I don't think this is used in the service, is it?

Reply:
It should be imported and used.